bin/hoc-2015-nov-9-mailing/send-general-final-real-mails

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+# ubuntu@production-console:~/production/bin/hoc-2015-nov-9-mailing$ wc -l general-deduped-remainder.csv
+# 1261412 general-deduped-remainder.csv
+# 1,261,412
+
+./send-to-mailing-list --start       0 --limit 100000 11-9-hoc-general hoc-2015-nov-9-mailing/general-deduped-test-remainder.csv
```

Comment:
oh, I see what you're saying. That part was off my screen; I just saw the comment above. Thanks. 

Fixed.